e2e/cypress/integration/enterprise/guest_accounts/guest_identification_ui_spec.js

```diff
@@ -234,6 +234,6 @@ describe('MM-18045 Verify Guest User Identification in different screens', () =>
         });
 
         // # Close and Clear the Search Autocomplete
-        cy.get('#searchClearButton').click();
+        cy.get('#searchFormContainer').find('.input-clear-x').click({force: true});
```

Comment:
Related to recent change where selector was removed but it's fine not to add it back.